ridemyway/api/vehicle/models.py

```diff
@@ -0,0 +1,16 @@
+from django.conf import settings
+from django.db import models
+
+from ridemyway.api.utilities.base_classes.models import BaseModel
+
+
+class Vehicle(BaseModel):
+    number_plate = models.CharField(max_length=20, unique=True)
```

Comment:
Hi, amazing work was done here. I'll suggest we add the `make` and `model` of the car to this model so that a user can know the `make` and probably the model of the car coming to pick them up just like the app `Uber` and `Bolt` does. Example of the `make` can be `Toyota` or `BMW` while the model can be `Camry` as regards to `Toyota` of `7 series` as regards to `BMW`.

Reply:
Done.